packages/api-reference/.storybook/preview.css

```diff
@@ -0,0 +1,61 @@
+body.sb-show-main,
```

Comment:
Nice added the theming!!! Glad to see its getting useage

Reply:
Maybe @cameronrohani can give storybook a look to see if everything is the right color, I just guessed

Reply:
Also maybe we should move this into  @scalar/themes and consume it from components and references.